extractor/src/main/scala/com/digitalasset/extractor/Extractor.scala

```diff
@@ -42,6 +46,7 @@ class Extractor[T](config: ExtractorConfig, target: T)(
     extends StrictLogging {
 
   private val tokenHolder = config.accessTokenFile.map(new TokenHolder(_))
+  private val parties: Set[String] = config.parties.toSet.map[String, Set[String]](identity)
```

Comment:
There is probably a way to do it statically, without the iteration, using `scalaz.Liskov` or `scalaz.Leibniz`... but I have not fully understood those yet :) so I will leave it for @S11001001 .... but in any case this is a constant.

Reply:
ha... I see the above thread.

Reply:
Yeah, there is `widen` as suggested by @S11001001 in an earlier comment. But unfortunately it triggers the wartremover `Any` check. Since it's only a single iteration over the parties it's not a big deal, albeit using `widen` would be more elegant of course.